oat-sa/extension-tao-itemqti
# fix/TAO-10387/mime-type-file-upload-extension

Related to : https://oat-sa.atlassian.net/browse/TAO-10387
 
update package.json
  
#### How to test

- aopen any test item for authoring
- add file upload interaction to item
- save item and open item preview
- ensure selected Pascal, C, CSV, FLS, etc mime types correctly used as file type filter
  
#### Dependencies
 
Requires :
 - [ ] https://github.com/oat-sa/tao-item-runner-qti-fe/pull/113
